packages/framework/ai-collab/src/explicit-strategy/typeGeneration.ts

```diff
@@ -17,7 +17,7 @@ import type {
 	SimpleTreeSchema,
 	TreeNode,
 } from "@fluidframework/tree/internal";
-import { z } from "zod";
+import { z, type ZodTypeAny } from "zod";
```

Comment:
I'm not sure why I have to make these changes in this PR. Something to do with how the package-lock is resolving the zod packages due to past version changes, even though I did not change any zod package versions with this PR. I'm okay with it since I think this is a slight improvement. Also we'll probably delete this whole package eventually anyway.